contracts/interchain-token-service/src/contract/migrations/v1_1_1.rs

```diff
@@ -79,7 +79,7 @@ pub fn migrate(deps: DepsMut, msg: MigrateMsg) -> Result<(), ContractError> {
 fn convert_max_uint_to_max_bits(max_uint: nonempty::Uint256) -> Result<NumBits, ContractError> {
     // Need to add one to correctly get the number of bits. This will round down if the result is not a power of two
     #[allow(clippy::arithmetic_side_effects)] // can't possibly overflow
-    NumBits::try_from(Uint512::from(*max_uint).ilog2() + 1).map_err(|err| err.into())
```

Comment:
This max uint to max bits migration was already done on mainnet, so you would require another migration

Reply:
removed